packages/api-generator/src/locale/en/v-overlay-location-strategies.json

```diff
@@ -3,6 +3,6 @@
     "locationStrategy": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-overlay-location-strategies.json))",
     "location": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-overlay-location-strategies.json))",
     "origin": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-overlay-location-strategies.json))",
-    "offset": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-overlay-location-strategies.json))"
+    "offset": "Offset the menu on both x-axis and y-axis. Works in conjunction with direction top/bottom and left/right direction, possible values are single number or [x,y] for example [10, 20]"
```

Comment:
Maybe this should be added to the upgrade guide as well?

Reply:
Yes, can you point me to the upgrade guide file , i can submit PR for it

Reply:
"on both x-axis and y-axis" isn't really correct. The first value is distance from the activator side in `location`, second value shifts along that side.